### DISCUSSION TITLE
v1.8.4 Rpi 0wh waveshare v4

### DISCUSSION THREAD

**Discussion by OutsiderSV** [aluminum-ice/pwnagotchi#103] [General]
I just flashed my sd card with version 1.8.4 and it boots up I can ssh into it
when I try to turn it on into auto mode it freezes on the hack the planet initializing image and does not refresh
then it goes into Ai ready but never starts searching or scanning wifi's

**Comment by OutsiderSV** [comment]
> I just flashed my sd card with version 1.8.4 and it boots up I can ssh into it when I try to turn it on into auto mode it freezes on the hack the planet initializing image and does not refresh then it goes into Ai ready but never starts searching or scanning wifi's

I get this message in the logs
[2024-01-12 23:08:35,677] [ERROR] error connecting to the pwngrid-peer service: HTTPConnectionPool(host='127.0.0.1', port=8666): Max retries exceeded with url: /api/v1/data (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0xaffe8850>: Failed to establi$[

**Comment by wpa-2** [comment]
That's because this REPO doesnt support the pizeroWH 
Use this image
https://github.com/wpa-2/pwnagotchi/releases/tag/Fix
